js/adapt-contrib-bookmarking.js

```diff
@@ -49,7 +96,7 @@ class Bookmarking extends Backbone.Controller {
     _.delay(() => {
       if (this.isAlreadyOnScreen(this.restoredLocationID)) return;
       if (Adapt.course.get('_bookmarking')._showPrompt === false) {
-        this.navigateToPrevious();
+        this.navigateTo();
```

Comment:
The button is there as a supplementary feature and not as a mandatory click for bookmarking. And it is up to the design to add the button or not.

I don't see a problem with the current implementation. If it is specifically decided to remove the popup on course load, the course will still follow the prescribed settings to navigate to the previous or furthest point, even if a button exists.

Reply:
Yea, it's just a use case which feels relatively intuitive to me.

I have a feeling it'd be nicer to land on the menu and be able to optionally resume rather than just to jumping straight in. This is effectively what the popup allows you to do, either resume or dismiss. It would be nice to have that similar flow without the popup. I imagine the resume button to be used most often on a menu.